sharktank/sharktank/utils/iree.py

```diff
@@ -112,6 +112,22 @@ def f(
         return f
 
 
+def get_file_handle(
+    weight_path: Path,
+    shard_count: int = 1,
+) -> List[FileHandle]:
+    handles = []
+    modified_weight_path = weight_path
+    for i in range(shard_count):
+        if shard_count > 1:
+            modified_weight_path = str(
+                weight_path.with_suffix(f".rank{i}{weight_path.suffix}")
+            )
+        with open(str(modified_weight_path), "rb") as f:
+            handles.append(FileHandle.wrap_fd(f.fileno(), True, False))
```

Comment:
You could skip the `FileHandle` default args or use named parameters. Just `True, False` is not that informative.